bin/tronctl

```diff
@@ -147,12 +179,50 @@ def control_objects(args):
             )
             raise SystemExit(f"Error: {e}{suggestions}")
 
-        data = dict(command=args.command)
-        if args.command == "start" and args.run_date:
-            data['run_time'] = str(args.run_date)
-        if args.command == "retry":
-            data['use_latest_command'] = int(args.use_latest_command)
-        yield request(urljoin(args.server, tron_id.url), data)
+        if args.command == "skip-and-publish":
+            # this command is more of a pseudo-command - skip and publish are handled in two different resources
+            # and changing the API would be painful, so instead we call skip + publish separately from the client
+            # (i.e., this file) to implement this functionality
+            if request(
+                url=urljoin(args.server, tron_id.url),
+                data={"command": "skip"}
+            ):
+                # a single action can have 0..N triggers to publish and these can be arbitrarily named, so we need to
+                # query the API and figure out what triggers exist
+                triggers = _get_triggers_for_action(server=args.server, action_identifier=identifier)
+                if triggers is None:
+                    # tronctl allows for multiple ids to be passed in for commands, but we want to make it obvious that
```

Comment:
I'm not sure what decision you're explaining with this comment - is it the `yield False`, so it exits early?

Reply:
yup - same with the comment below. do you think moving the comment(s) so that the yield(s) follow immediately after would make that clearer? open to suggestions :)

Reply:
I don't think you need to necessarily justify it separately, because other operations also yield the result of `request` after every ID, and could fail. (I suppose you changed the behavior of request to make it possible to fail in this PR, but I think exiting early makes sense in general.)

Reply:
ah cool - I'll remove these comments then

Reply:
done!